content/docs/sidebar.json

```diff
@@ -91,6 +91,12 @@
         "label": "What is DVC?",
         "slug": "what-is-dvc"
       },
+      {
+        "label": "Basic Concepts",
+        "slug": "basic-concepts",
```

Comment:
Should we just call the section "Concepts"?

Reply:
my 2cs - `basic concepts` sounds more familiar